packages/@aws-cdk/aws-codepipeline-actions/lib/bitbucket/source-action.ts

```diff
@@ -98,6 +104,14 @@ export class BitBucketSourceAction extends Action {
     // the action needs to write the output to the pipeline bucket
     options.bucket.grantReadWrite(options.role);
 
+    // if codeBuildCloneOutput is true,
+    // save the connectionArn in the Artifact instance
+    // to be read by the CodeBuildAction later
+    if (this.props.codeBuildCloneOutput === true) {
```

Comment:
Why not always do that?

Reply:
Because if this property is on, no Action other than the CodeBuild one can use the artifact produced by this Action, which I don't think is what we want.